src/OpenTracing/LogFields.cs

```diff
@@ -1,45 +1,41 @@
-﻿using System;
-
-namespace OpenTracing
+﻿namespace OpenTracing
 {
     /// <summary>
-    /// <para>Every Span log has a specific timestamp and one or more fields.
-    /// This class defines the standard fields.</para>
-    /// <para>See https://github.com/opentracing/specification/blob/master/semantic_conventions.md for details.</para>
+    /// The following log fields are recommended for instrumentors who are trying to capture more information about a
+    /// logged event. Tracers may expose additional features based on these standardized data points.
+    /// <see href="https://github.com/opentracing/specification/blob/master/semantic_conventions.md"/>
     /// </summary>
     public static class LogFields
     {
-        /// <summary>
-        /// The type or "kind" of an error (only for event="error" logs).
-        /// <para />E.g., "Exception", "OSError".
-        /// </summary>
+        /// <summary>The type or "kind" of an error (only for event="error" logs). E.g., "Exception", "OSError"</summary>
         public const string ErrorKind = "error.kind";
 
         /// <summary>
-        /// The actual <see cref="Exception"/> object instance.
+        /// The actual Throwable/Exception/Error object instance itself. E.g., A java.lang.UnsupportedOperationException
```

Comment:
I think the existing comment (`/// The actual <see cref="Exception"/> object instance.`) makes more sense here since it is a better explanation for .NET users and doesn't mention a java type.

Reply:
Will change but want to bring onto your radar my logic here:

I was concerned about that since tracing is distributed. I do not want to suggest that it will always actually be an Exception as if the distributed system has multiple languages it might not be.

However, this is log and not Baggage/Tag, and while it's possible an implementation put all it's Logs as Baggage (some... very test-oriented implementation :) ) in retrospect it's unlikely this would be a problem.